cluster/node-pools/master-default/userdata.yaml

```diff
@@ -853,6 +859,25 @@ write_files:
           - identity: {}
       {{- end }}
 
+  - owner: root:root
+    path: /etc/kubernetes/config/scheduler-config.yaml
+    permissions: '0400'
+    content: |
+      apiVersion: kubescheduler.config.k8s.io/v1
+      kind: KubeSchedulerConfiguration
+      clientConnection:
+        kubeconfig: /etc/kubernetes/scheduler-kubeconfig
+      profiles:
+        - schedulerName: default-scheduler
+          pluginConfig:
+            - name: PodTopologySpread
+              args:
+                defaultConstraints:
+                  - maxSkew: 1
+                    topologyKey: topology.kubernetes.io/zone
+                    whenUnsatisfiable: ScheduleAnyway
+                defaultingType: List
```

Comment:
Let's put it behind a config-item. Can be on by default but let us have a quick way to turn it off if it's not behaving as expected.